src/govuk/components/cookie-banner/template.test.js

```diff
@@ -0,0 +1,219 @@
+
+      expect($banner.hasClass('app-my-class')).toBeTruthy()
+    })
+
+    it('renders attributes', () => {
+      const $ = render('cookie-banner', examples.attributes)
+
+      const $banner = $('.govuk-cookie-banner .govuk-cookie-banner__container')
+
+      expect($banner.attr('data-attribute')).toEqual('my-value')
+    })
+  })
+
+  describe('role and aria attributes', () => {
+    it('has role=region by default', () => {
+      const $ = render('cookie-banner', examples.default)
+
+      const $component = $('.govuk-cookie-banner')
+      expect($component.attr('role')).toEqual('region')
+    })
+
+    it('has a default aria-label', () => {
+      const $ = render('cookie-banner', examples.default)
+
+      const $component = $('.govuk-cookie-banner')
+      expect($component.attr('aria-label')).toEqual('Cookie banner')
+    })
+
+    it('renders a custom aria label', () => {
+      const $ = render('cookie-banner', examples['custom aria label'])
+
+      const $component = $('.govuk-cookie-banner')
+
+      // expect($component.attr('role')).toEqual('region')
+      expect($component.attr('aria-label')).toEqual('Cookies on GOV.UK')
+    })
+  })
+
+  describe('confirmation banner', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('cookie-banner', examples['accepted confirmation banner'])
+
+      const results = await axe($.html())
+      expect(results).toHaveNoViolations()
+    })
+
+    it('set role attribute to alert', () => {
+      const $ = render('cookie-banner', examples['accepted confirmation banner'])
+
+      const $component = $('.govuk-cookie-banner')
+      const $banner = $component.find('.govuk-cookie-banner__container')
+      expect($banner.attr('role')).toEqual('alert')
+    })
+
+    it('role alert not set by default', () => {
+      const $ = render('cookie-banner', examples.default)
+
+      const $component = $('.govuk-cookie-banner')
+      const $banner = $component.find('.govuk-cookie-banner__container')
+      expect($banner.attr('role')).toBeUndefined()
+    })
+
+    it('hides banner if hidden option set to true', () => {
+      const $ = render('cookie-banner', examples.hidden)
+
+      const $component = $('.govuk-cookie-banner__container')
+      expect($component.attr('hidden')).toBeTruthy()
+    })
+
+    it('does not hide banner if hidden option set to false', () => {
+      const $ = render('cookie-banner', examples['hidden false'])
+
+      const $component = $('.govuk-cookie-banner__container')
+      expect($component.attr('hidden')).toBeUndefined()
+    })
+  })
+
+  describe('action', () => {
+    it('renders as button by default', () => {
+      const $ = render('cookie-banner', examples['default action'])
+
+      const $actions = $('.govuk-cookie-banner .govuk-button')
+      expect($actions.get(0).tagName).toEqual('button')
+      expect($actions.text().trim()).toEqual('This is a button')
```

Comment:
There's a few tests in this describe block that have multiple expectations, and they don't always directly relate to the test description. 

Are some of these expectations repeated? Could any of them be removed, or move into their own tests?